### PR TITLE
KEYCLOAK-5289 Update documentation for the Google identity provider config page

### PR DESCRIPTION
**Pull request by brushmate**

### CODE REVIEW

**Comment by brushmate** [comment]
Keycloak PR: keycloak/keycloak#5215

File: server_admin/topics/identity-broker/social/google.adoc [hunk 1]
```diff
@@ -81,3 +81,6 @@ One config option to note on the `Add identity provider` page for Google is the
 This field allows you to manually specify the scopes that users must authorize when authenticating with this provider.
 For a complete list of scopes, please take a look at https://developers.google.com/oauthplayground/ . By default, {project_name}
 uses the following scopes: `openid` `profile` `email`.
+
+If your organization uses the G Suite, you might want to restrict access to members of your organization. You have to
+enter the domain that is used for the G Suite into the `Hosted Domain` field to enable it.
```
Comment (matthewhelmke) [inline server_admin/topics/identity-broker/social/google.adoc]: If your organization uses the G Suite and you want to restrict access to only members of your organization,  you must enter the domain that is used for the G Suite into the `Hosted Domain` field to enable it.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Thank you. I left a review comment with some minor changes that I am requesting.

**Comment by brushmate** [comment]
I updated the PR according to your comment.

**Review by matthewhelmke — APPROVED** [review]

**Comment by matthewhelmke** [comment]
Thank you!

**Pull request merged**